monitoring/monitoring_internal_test.go

```diff
@@ -181,528 +192,543 @@ func TestPeriodicReport(t *testing.T) {
 }
 
 func TestSystemAlerts(t *testing.T) {
-	sendDuration := 1 * time.Second
+	duration := 100 * time.Millisecond
 
-	alertMap := make(map[string]int)
+	var alertMap []cloudprotocol.AlertItem
```

Comment:
It is not a map anymore. Could you rename it?

Reply:
Done